test/ext/asyncio/test_engine_py3k.py

```diff
@@ -590,6 +591,15 @@ async def test_create_async_engine_server_side_cursor(self, async_engine):
             server_side_cursors=True,
         )
 
+    def test_async_engine_from_config(self):
+        config = {
+            "sqlalchemy.url": str(testing.db.url),
```

Comment:
Create async engine will raise IIRC if the current dialect is not async. It's probably best to hardcore it to aiosqlite

Reply:
Nevermind it's in the async test so it should be fine.